neptune-export/src/main/java/com/amazonaws/services/neptune/export/ExportToS3NeptuneExportEventHandler.java

```diff
@@ -104,6 +106,7 @@ public static ObjectTagging createObjectTags(Collection<String> profiles) {
     private final Collection<String> profiles;
     private final Collection<CompletionFileWriter> completionFileWriters;
     private final AtomicReference<S3ObjectInfo> result = new AtomicReference<>();
+    private static final Pattern STATUS_CODE_5XX_PATTERN = Pattern.compile("Status Code: (5\\d+)");
```

Comment:
Is it always trying to match 3 digit 5XX status codes? Could change regex to `"Status Code: (5\\d\\d)"` to be a bit more specific.

Reply:
There should only be 3 digit 5xx status code, so I think either works?